### DISCUSSION TITLE
How to Impersonate an account with Brownie?

### DISCUSSION THREAD

**Discussion by ghost** [smartcontractkit/full-blockchain-solidity-course-py#1810] [Q&A] [answered]
Hello,

I have forked ETH mainnet and would like to imperonate a wallet that has sufficient funds (similar to hardhat). Could anyone advise on how to do this with Brownie? I'm having a hard time finding much info on this for Brownie. 

- Thank you

**Comment by ghost** [comment, marked as answer]
Nevermind, I was able to find it in the documentation.

**Reply by cromewar** [reply]
Nice catch @Dee239, feel free to make any other question.
